app/utilities/metadata_parser.py

```diff
@@ -88,6 +98,11 @@ class RunnerMetadataSchema(Schema, StripWhitespaceMixin):
         required=False, validate=validate.Length(min=1)
     )  # type:ignore
     case_type = VALIDATORS["string"](required=False)  # type:ignore
+    response_expires_at = VALIDATORS["iso_8601_date_string"](
+        required=False,
+        validate=lambda x: datetime.strptime(x, ISO_8601_DATETIME).astimezone()
```

Comment:
`astimezone()` without a timezone will use the systems' timezone which we can't garuntee is UTC. So we need to either use `.replace(tzinfo=timezone.utc)` or `.astimezone(tz=timezone.utc)`, we use `.replace()` elsewhere prob cause the time data shouldn't change.